forcats.Rmd

```diff
@@ -82,8 +82,152 @@ factor_clarity %>% fct_unique()
 factor_clarity %>% unique()
 ```
 
-
 ## Combine factors
+### Combine factors with different levels
+With the function ***fct_c()*** factors with different levels can be combined to one factor covering all levels. First we create two factors. The first one shows the amount of contested disciplines by several countries during the olympic sommer games 1896. The second one shows the amount of contested disciplines by countries during the olympic winter games 1924.
+
+```{r}
+olympic_1896 <- olympic %>% filter(game == "1896 Summer")
+summer_1896 <- factor(olympic_1896$country)
+olympic_1924 <- olympic %>% filter(game == "1924 Winter")
+winter_1924 <- factor(olympic_1924$country)
+```
+
+The factors differ both in their stored categorical data and in their set levels. A plot makes this clear:
+
+```{r}
+summer_1896 %>%
+  fct_count() %>%
+  ggplot(aes(`amount of disciplines`, country))+
+  geom_col(aes(n,f))
+
+winter_1924 %>%
+  fct_count() %>%
+  ggplot(aes(`amount of disciplines`, country))+
+  geom_col(aes(n,f))
+```
+
+Now we combine the factors with ***fct_c()*** and plot it:
+
+```{r}
+fct_c(summer_1896, winter_1924) %>%
+  fct_count() %>%
+  ggplot(aes(`amount of disciplines`, country))+
+  geom_col(aes(n,f))
+```
+
+Both the underlying categorical data and the defined levels were combined in one factor.
+
+### Standardise levels of various factors
+With the function ***fct_unify*** the levels of different factors can be standardised. Please note, both input and output are in list form.
+```{r}
+factor_list <- fct_unify(list(summer_1896, winter_1924))
+
+# plotting the first factor
+factor_list[[1]] %>%
+  fct_count() %>%
+  ggplot() + 
+  geom_col(aes(n, f))
+
+# plotting the second factor
+factor_list[[2]] %>%
+  fct_count() %>%
+  ggplot() + 
+  geom_col(aes(n, f))
+```
+
+In this case, the underlying categorical data was left unchanged for both factors, but the levels were standardised. This is especially useful when comparing the categorical data of two different factors. 
+
 ## Order of levels
+### Manual reordering of levels
+With the function ***fct_relevel*** the levels of a factor can be reordered. In contrast to the function ***levels()***, which only allows the renaming of factor levels, ***fct_relevel*** also adjusts the order of the levels themselves, that is the way they are stored internally. An example should clarify this.
+
+```{r}
+# have a look at the original levels
+factor_clarity %>% 
+  fct_count()
+# The wrong approach (this is semantical wrong):
+levels(factor_clarity) <- c("c1","c3","c5","c7","c2","c4","c6","c8")
+factor_clarity %>% 
+  fct_count()
+#tidy up
+levels(factor_clarity) <- levels_clarity_c
+# The right approach:
+factor_clarity %>%
+  fct_relevel(c("c1","c3","c5","c7","c2","c4","c6","c8")) %>%
+  fct_count()
+```
+
+As you can see, only the function ***fct_relevel*** allows a correct relevel.
+
+### Reordering by frequency
```

Comment:
I think we should check what `infreq` stands for and mention it. 
My first guess would be that it is short for <code>sort <b>in</b> order of <b>freq</b>uency </code>.